style.css

```diff
@@ -0,0 +1,806 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+  background-color: #f5f5f5;
+  color: #272a31;
+}
+
+hr {
+  width: 35px;
+}
+
+.mobile-menu {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  background-color: #b1b1b1;
+  top: -110%;
+  left: 0;
+  z-index: 100;
+  transition: 500ms ease-in-out;
+  text-align: center;
+}
+
+.mobile-menu ul {
+  width: 90%;
+  margin: 0 auto;
+  margin-top: 100px;
+}
+
+.logout-nav li,
+.nav-with-image li {
+  display: inline-block;
+  margin-right: 30px;
+  list-style: none;
+}
+
+.mobile-menu ul li {
+  position: relative;
+  list-style: none;
+  margin: 35px auto;
+  z-index: 999;
+}
+
+.program_button a {
+  display: none;
+}
+
+.time_venu a {
+  color: #1e2022;
+  font-weight: bolder;
+  transition: 200ms;
+}
+
+.logout-nav li a {
+  color: #d3d3d3;
+  font-size: 12px;
+  text-decoration: none;
+  transition: 400ms;
+}
+
+.nav-with-image li a {
+  color: #272a31;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+.mobile-menu ul li a {
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 32px;
+  color: #272a31;
+  text-decoration: none;
+  padding: 5px;
+  transition: 300ms ease-in-out;
+  cursor: pointer;
+}
+
+.hamburger i {
+  cursor: pointer;
+  transition: 200ms;
+  transform: scale(0.98);
+}
+
+.program_button a:hover,
+.time_venu a:hover,
+.mobile-menu ul li a:hover {
+  color: #ec5242;
+  transform: scale(1.03);
+}
+
+.festival_programs i,
+.festival_programs h4 {
+  align-self: center;
+  justify-self: center;
+}
+
+.festival_programs i {
+  color: #fff;
+  cursor: pointer;
+  transition: 300ms ease-in;
+}
+
+.go-up i {
+  color: #e8240e;
+}
+
+.time_venu h2 i,
+.time_venu h3 i {
+  margin-right: 10px;
+}
+
+.time_venu p i {
+  margin-right: 20px;
+}
+
+.program:hover i,
+#main-nav li a:hover,
+.hamburger i:hover {
+  color: #ec5242;
+}
+
+.mobile-menu img {
+  position: absolute;
+  top: 25px;
+  right: 30px;
+  z-index: 3;
+  cursor: pointer;
+}
+
+footer .footer_copy img {
+  width: 50px;
+  height: 50px;
+}
+
+.mobile-menu img:hover {
+  animation: rotateX 300ms ease-in-out;
+  color: #ec5242;
+}
+
+.hamburger {
+  width: 80%;
+  margin: 0 auto;
+  height: 50px;
+  display: flex;
+  align-items: center;
+}
+
+.guest_list .guest img {
+  position: relative;
+  width: 120px;
+  z-index: 3;
+}
+
+#main-nav .nav-with-image {
+  height: 70px;
+  width: 80%;
+  margin: 0 auto;
+  background-color: #f5f5f5;
+  display: none;
+}
+
+.logout-nav {
+  background-color: #272a31;
+  height: 30px;
+  display: none;
+}
+
+.logout-nav nav {
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+#main-nav .nav-with-image img {
+  width: 80px;
+  padding-top: 10px;
+}
```

Comment:
- [ ] Logo overflowing the container. It will be better if you set height for the image instead of width. Since you set the height of the parent container to 80px. 
![Screenshot 2021-08-14 at 13-00-34 International Festival Theatre in KUMASI](https://user-images.githubusercontent.com/1642014/129442563-106ff6ba-382a-450b-aec1-bc98bc482695.png)
```suggestion
#main-nav .nav-with-image img {
  height: 60px;
  padding-top: 10px;
}
```